.travis.yml

```diff
@@ -13,16 +13,33 @@ addons:
             - liblapack-dev
 
 cache:
-    apt: true
-    pip: true
-    ccache: true
+    apt: true   # does not work for public repos
+    directories:
+        - $HOME/.cache/pip
+        - $HOME/.ccache
+        - $HOME/nltk_data
+        - $TRAVIS_BUILD_DIR/pyqt
+
+before_cache:   # prevent logs from caching
+    - rm -f $HOME/.cache/pip/log/debug.log
 
 before_install:
+    - source $TRAVIS_BUILD_DIR/.travis/util.sh
     - pip install -U setuptools pip wheel
     - pip install codecov
+    # Gensim (2.2.0) requires numpy>=1.11.3 but cannot update it itself?
+    - pip install -U numpy
+    - mkdir -p /home/travis/.local/share/Orange  # create orange app dir
 
 install:
     - travis_wait pip install -e .
+    - source $TRAVIS_BUILD_DIR/.travis/install_pyqt.sh
```

Comment:
No. Just `pip install PyQt5 AnyQt` if they aren't specified as dependencies for some reason. Should pip-install cleanly on Python 3.5.